index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name= "viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <link rel="stylesheet" type="text/css" href="styles/normalize.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Lora:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <title>Adopt a Dog - Home</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-top">
+        <div class="logo-bundle">
+          <img class="header-logo" src="images/adoptadog-logo.svg" alt="Adopt a Dog Logo">
+          <p>Seattle's premiere dog adoption agency</p>
+        </div>
+        <div class="shopping-cart">
+          <img class="shopping-cart-icon" src="images/cart-icon.svg" alt="Shopping cart icon">
+          <p>$1234.56</p>
+          <button class="cart-button" type="button">Checkout</button>
+        </div>
+      </div>
+      <div>
+        <ul class="header-nav">
+          <li><a href="index.html">Home</a></li>
+          <li><a href="our-dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </div>
+    </header>
+    <main>
+      <section>
```

Comment:
You will want to wrap your dog tiles in div tags, not sections. You might use a section to wrap all of your dog-tiles into a container, but div tags should be used for the individual tiles. Also, you'll want to put some styling in. When I visit your site the dog images take up the entire page. See if you can add a css class to these dog tiles to bring them down in size. Then wrap these dog tiles in a section or div, and apply a css class to that to create the grid appearance.